src/INode.java

```diff
@@ -189,9 +189,29 @@ private ASTValue evalAnd(WordsEnvironment environment) {
 		throw new AssertionError("Not yet implemented");
 	}
 
-	private ASTValue evalAssign(WordsEnvironment environment) {
-		// TODO
-		throw new AssertionError("Not yet implemented");	
+	private ASTValue evalAssign(WordsEnvironment environment) throws WordsRuntimeException {
+		WordsObject obj = children.get(0).eval(environment).objValue;
```

Comment:
This isn't ever making a call to the environment to get an object. Does this work? I think that the 0th child is actually storing the object name.